examples/components/data_table/selectable_usage.html

```diff
@@ -17,7 +17,7 @@
   </tr>
   </thead>
   <tbody>
-  <tr *ngFor="#material of materials"  md-data-table-selectable-row [selectable-value]="material.id">
+  <tr *ngFor="#material of materials"  md-data-table-selectable-row [selectable-value]="material.id" [is-active]="material.id == 2">
```

Comment:
This shouldn't be a part of this example in my opinion. The doc is enough for me